securedrop/source_templates/base.html

```diff
@@ -3,8 +3,9 @@
   <head>
     <meta charset="utf-8">
     <title>SecureDrop | Protecting Journalists and Sources</title>
-    {% assets filters="cssmin", output="gen/source.css",
-      "css/normalize.css", "css/source.css", "css/font-awesome.css" %}
+
+    {# TODO why doesn't a link to /static/source.css work? #}
```

Comment:
I'll take a look at this since there have been some updates to some of the dependencies since I wrote this and the behavior might have changed. Also, suggestions welcome since this one seems a bit non-intuitive.